reference-architecture/aws-ansible/playbooks/roles/s3-registry-bucket/tasks/main.yaml

```diff
@@ -1,6 +1,10 @@
 ---
 - name: set the name of the bucket
-  set_fact: s3_bucket="{{ ansible_date_time.epoch }}-openshift-docker-registry"
+  set_fact: s3_bucket="{{ ansible_date_time.epoch }}-openshift-docker-registry-{{ stack_name}}"
```

Comment:
why the epoch for the s3 bucket here? Isn't that going to make it so that if this role is re-applied that we are going to create a new s3 bucket each time?